catalog-info.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: upperkut
+  description: Ferramenta de processamento em segundo plano para Ruby.
+  annotations:
+    github.com/project-slug: ResultadosDigitais/upperkut
```

```suggestion
  annotations:
    circleci.com/project-slug: github/ResultadosDigitais/upperkut
    github.com/project-slug: ResultadosDigitais/upperkut
```
Esse repo tá ativo no circleci também, então dá pra colocar a annotation https://app.circleci.com/pipelines/github/ResultadosDigitais/upperkut?filter=all

---

aaa boa, valeu!!